apps/artemis-web/scripts/query.js

```diff
@@ -47,5 +47,7 @@ async function isInCollection(db, name) {
     }
   } catch (e) {
     console.error(e);
+  } finally {
+    client.close();
```

Comment:
This will revert the change I have made, but will re-introduce the bug with the authentication error we had.

Give me a couple of hours to see if we can fix both.